test/remote/gateways/remote_sum_up_test.rb

```diff
@@ -51,6 +51,16 @@ def test_successful_purchase_with_more_options
     assert_equal 'PAID', response.message
   end
 
+  def test_successful_purchase_with_partner_id
+    options = {
+      partner_id: 'PartnerId',
+      order_id: SecureRandom.uuid
+    }
+
+    response = @gateway.purchase(@amount, @credit_card, options)
+    assert_equal "#{options[:partner_id]}-#{options[:order_id]}", response.params['checkout_reference']
```

Comment:
Could you include the
 ```ruby 
assert_success response
```
 to ensure this will be a success transaction with this field?

Reply:
I didn't because the gateway transactions are failing and it wasn't quite clear why. Since nothing has changed that would break the integration, I oped to ignore it for now.